pkg/agent/flannel/setup.go

```diff
@@ -86,12 +86,13 @@ func Run(ctx context.Context, nodeConfig *config.Node, nodes v1.NodeInterface) e
 			break
 		}
 		if err == nil {
-			logrus.Infof("waiting for node %s CIDR not assigned yet", nodeName)
+			logrus.Infof("Waiting for node " + nodeName + " CIDR not assigned yet")
```

Comment:
Unclear on why string concatenation is better than formatting when we're still using Infof() instead of Info(). For consistency with the else statement, why not just do:
 
```suggestion
			logrus.Infof("Waiting for node %s: CIDR not assigned yet")
```